packages/website/docs/Features/Language.mdx

```diff
@@ -1,39 +1,51 @@
 ---
 sidebar_position: 2
+title: Language Basics
```

Comment:
The writing of this document seems geared toward people with experience with language design. However, I expect the readers to not be so familiar. 

For example,
"squiggle buffer should be an expression"
"A squiggle **record** is a key-value store with dot accessors"

This is still better than having less documentation, but I think could be improved.

Reply:
totally revamped